deploy/backplane/acs/01-acs-admins-cluster-aggregate.ClusterRole.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-acs-admins-cluster-aggregate
+  labels:
+    managed.openshift.io/aggregate-to-acs-admins: "cluster"
+rules:
+# ACS Eng can manage projects
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - projects
+  verbs:
+  - get
+  - list
+# ACS SRE can get infrastructure details (cluster)
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - infrastructures
```

Comment:
Just a note that `infrastructures` is not listed in the requirements.  I have no problem with it being granted (i.e. no action required), maybe it is something we can call out as "standard" view permissions that anybody gets via backplane sometime.

Reply:
I will add it to the service requirements doc